features/step_definitions/log_in_steps.rb

```diff
@@ -7,11 +7,9 @@
 end
 
 And(/^I submit login form with valid data$/) do
-  login = @user[:login]
-  password = @user[:password]
-  visit(LoginPage).log_in(login,password)
+  visit(LoginPage).log_in(@user[:login],@user[:password])
```

Comment:
note that you can pass only ```@user``` here and get its login and password inside a method
this is a good practice btw